lib/src/delegate/future.dart

```diff
@@ -25,13 +25,13 @@ class DelegatingFuture<T> implements Future<T> {
   Stream<T> asStream() => _future.asStream();
 
   Future<T> catchError(Function onError, {bool test(Object error)}) =>
-    _future.catchError(onError, test: test);
+      _future.catchError(onError, test: test);
 
-  Future/*<S>*/ then/*<S>*/(/*=S*/ onValue(T value), {Function onError}) =>
-    _future.then(onValue, onError: onError);
+  Future/*<S>*/ then/*<S>*/(dynamic onValue(T value), {Function onError}) =>
```

Comment:
What SDK version did this change in? We should set the minimum SDK constraint to that version.